### ISSUE TITLE
UWP Startup Hangs

### ISSUE THREAD

**Issue by elucidsoft** [MvvmCross/MvvmCross#3221]
This is still occurring with latest framework. I can do a Google Hangout with anyone who wants to painfully watch me recreate it. But basically shutdown and restart the app in succession, its taken me up to 30 attempts but I can get it to happen. Seems to occur at a low frequency so it's very difficult to track down, does not seem to have as anything to do with resume or suspend. It appears to be a race condition as there is a Wait metamaphore its hung up on...

**Comment by nickrandolph** [comment]
Are you calling an await inside the Initialise task for your first view model?

**Comment by elucidsoft** [comment]
Yes if I dont the db wont be finished and the viewmodel will fail to load the data. Like I said I have a workaround but using await is most likely going to be a pretty recurrent theme for most people.
